packages/scss/src/utilities/animations/_sliding.scss

```diff
@@ -1,7 +1,7 @@
 @keyframes slideIn {
```

Comment:
slideIn seems to be a shortcut for slideInTop. is it on purpose ?

Reply:
by that i mean we got
- slideIn
- slideInLeft
- slideInRight
- slideInBottom

but no slideInTop

Reply:
Yes, as Top was seen as default